### DISCUSSION TITLE
roadrunner and frankenphp

### DISCUSSION THREAD

**Discussion by ziat1988** [roadrunner-server/roadrunner#1797] [General]
I recently found out there are a new frankenphp buit with Go and at the same time i see there are roadrunner. I get a little bit confused. 
Can anyone can clarify these two ? Both are for same purpose or they can be used in same project?
Thanks for help.

**Comment by wolfy-j** [comment]
Hi, 

RoadRunner is production ready and much faster in worker mode (which it was build for), it has a ton of plugins and can supervise not only HTTP but also pub sub, queues, grpc and much more.

FrankenPHP can run Wordpress (RR can not, yet). 

That’s about it.
